### PR TITLE
refactor: Updated vector search langchain tutorial with latest API changes

### PR DESCRIPTION
**Pull request by Adi8885**
# Description

This PR addresses
1. Updated [question_answering_documents_langchain.ipynb](https://github.com/GoogleCloudPlatform/generative-ai/blob/main/language/use-cases/document-qa/question_answering_documents_langchain.ipynb) tutorial with latest API changes as per latest release from [langchain-google-vertexai](https://pypi.org/project/langchain-google-vertexai/)

2. Added self as contributor to .github/CODEOWNERS

### CODE REVIEW

File: .github/CODEOWNERS
Comment (holtskinner) [inline .github/CODEOWNERS:1]: @Adi8885 You'll need to join the GoogleCloudPlatform organization at go/github for this to work.

Comment (Adi8885) [reply .github/CODEOWNERS:1]: @holtskinner Requested Access, awaiting grant

**Review by holtskinner — CHANGES_REQUESTED** [review]
Remove project ID

File: language/use-cases/document-qa/question_answering_documents_langchain_matching_engine.ipynb [hunk 1]
```diff
@@ -52,7 +52,9 @@
   },
```
[the remark this answers is not in the document]
Comment (Adi8885) [reply language/use-cases/document-qa/question_answering_documents_langchain_matching_engine.ipynb:52]: Done

**Review by holtskinner — APPROVED** [review]

**Pull request merged**